lib/slim/embedded.rb

```diff
@@ -96,11 +96,11 @@ def initialize(opts = {})
       @disabled = normalize_engine_list(options[:disable_engines])
     end
 
-    def on_slim_embedded(name, body)
+    def on_slim_embedded(name, body, attrs)
       name = name.to_sym
       raise(Temple::FilterError, "Embedded engine #{name} is disabled") unless enabled?(name)
       @engines[name] ||= self.class.create(name, options)
-      @engines[name].on_slim_embedded(name, body)
+      @engines[name].on_slim_embedded(name, body, attrs)
```

Comment:
I've removed the `nil` check here, but this requires that all the engines' `on_slim_embedded` methods take an `attrs` param.

One alternative is to do something like this:
```rb
if @engines[name].is_a?(TagEngine)
  @engines[name].on_slim_embedded(name, body, attrs)
else
  @engines[name].on_slim_embedded(name, body)
end
```